### ISSUE TITLE
jx-convert-jenkinsfile fail on credentials in the environment

### ISSUE THREAD

**Issue by vbehar** [jenkins-x/jx#6522]
### Summary 

[jx-convert-jenkinsfile](https://github.com/jenkins-x/jx-convert-jenkinsfile) fails on a `Jenkinsfile` with an environment variable defined from a `credentials`.

### Steps to reproduce the behavior

Run version 0.0.1 on the following `Jenkinsfile`:

```
pipeline {
    environment {
        CREDS = credentials('something')
    }
}
```

### Expected behavior

no error

### Actual behavior

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x162aea6]

goroutine 1 [running]:
github.com/jenkins-x/jx-convert-jenkinsfile/pkg/grammar.(*ModelEnvironmentEntry).ToJXEnv(0xc0001c5740, 0xc0001c5b20, 0x20, 0x20, 0x168a880)
	/workspace/source/pkg/grammar/grammar.go:382 +0xd6
github.com/jenkins-x/jx-convert-jenkinsfile/pkg/grammar.toEnvYamlLines(0xc00000e140, 0x1, 0x1, 0x1, 0x2, 0xc0001c5b20, 0x1, 0x2)
	/workspace/source/pkg/grammar/grammar.go:320 +0xb7
github.com/jenkins-x/jx-convert-jenkinsfile/pkg/grammar.(*Model).ToYaml(0xc0001c5620, 0x1, 0xc0001c5620, 0x0, 0x0, 0x0)
	/workspace/source/pkg/grammar/grammar.go:128 +0x10f
main.main()
	/workspace/source/cmd/jx-convert-jenkinsfile/jx-convert-jenkinsfile.go:26 +0x1c5
```

**Comment by abayer** [comment]
Dang, thought I'd addressed this, but I missed something.

**Comment by abayer** [comment]
Fixed in v0.0.2, I hope. =)
